### ISSUE TITLE
How can I get the path to best .pth  file?

### ISSUE THREAD

**Issue by lilium513** [KevinMusgrave/pytorch-metric-learning#439]
Hello, thanks for this useful library!

I have a small question to use this library.

I'd like to train and inference at once.
Therefore, after training, I want to know path to the best weight( trunk_xxbest.pth) by calling some function.
Is there any function or way to get path to best model weight ?
If exist, could you tell me the function? (Then, really sorry for my overlooking)

I'll use the  _getBestModel_ function like
```python

# I can get the path to the best weight
path_to_best_trunk_model_weight = trainer.getBestModel()

# load best weight to inference
my_model.load_state_dict(path_to_best_trunk_model_weight)

```

Again, thank you for developing this great library !

**Comment by KevinMusgrave** [comment]
Hmm I think this should work:

```python
from pytorch_metric_learning.utils import common_functions as c_f

_, suffix = c_f.latest_version(model_folder, best=True)
best_path = os.path.join(model_folder, f"trunk_{suffix}.pth")
my_model.load_state_dict(best_path)
```

**Comment by lilium513** [comment]
Thank you for your Prompt answer. Your code gave me the  path to best weight ！
Thank you very much！！
